app/templates/survey/2022.hbs

```diff
@@ -1,110 +1,162 @@
+{{! END RESULTS SECTIONS }}
 
-    <p class="mt-4 text-sm">
-      Photos by <a href="http://www.octolabs.com/" rel="noopener noreferrer" target="_blank">Jeremy Green</a>
-    </p>
-  </:body>
-</EmberCommunitySurvey::Section>
 
+{{! CLOSING THOUGHTS / SPONSOR THANKS / PROMOTIONAL SHARE  }}
 <EmberCommunitySurvey::Section
-  @sectionId="spread-the-word"
-  @sectionTitle="Spread the Word!"
+  @sectionId='closing-thoughts'
+  @sectionTitle='Closing Thoughts'
 >
   <:body>
     <p>
-      Please share this survey on social media and at work.
+      We would like to thank everyone who took the time to participate in the
+      2022 Ember Community Survey! We genuinely appreciate the time and the valuable feedback our users provide for us every year. We also want to thank our sponsors for their generous support, making this year's survey possible.
     </p>
-
-    <div class="my-3">
+    <p>
+      To see results for previous years, you can visit the
+      <LinkTo @route='survey.2020'>2020</LinkTo>,
+      <LinkTo @route='survey.2019'>2019</LinkTo>,
+      <LinkTo @route='survey.2018'>2018</LinkTo>,
+      <LinkTo @route='survey.2017'>2017</LinkTo>,
+      <LinkTo @route='survey.2016'>2016</LinkTo>, and
       <a
-        class="twitter-share-button"
-        data-hashtags="EmberJS"
-        data-show-count="false"
-        data-size="large"
-        data-text="Do you use @emberjs? It&#39;s time for the 2022 survey!"
-        data-url="https://emberjs.com/survey/2022"
-        href="https://twitter.com/share?ref_src=twsrc%5Etfw"
-      >
-        Tweet
-      </a>
-
-      {{!-- template-lint-disable "no-forbidden-elements" --}}
-      <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
-    </div>
-
+        href='https://www.201-created.com/ember-community-survey-2015/'
+        rel='noopener noreferrer'
+        target='_blank'
+      >2015</a>
+      survey results pages.
+    </p>
     <p>
-      Questions? Feedback? Please join us in the <code>#dev-ember-learning</code> channel on the Discord community chat, or email the survey team via <a href="mailto:survey@emberjs.com">survey@emberjs.com</a>.
+      If you have any questions about this survey—the data, the methods used, or
+      any other feedback—please email us at
+      <a href='mailto:survey@emberjs.com'>survey@emberjs.com</a>.
     </p>
+    {{!-- <p>
+      Please share this survey on social media and at work.
+    </p> --}}
+
+
     <p>
-      See you at <a href="https://emberconf.com/" rel="noopener noreferrer" target="_blank">EmberConf 2022</a>!
+      Questions? Feedback? Please join us in the
+      <code>#dev-ember-learning</code>
+      channel on the Discord community chat, or email the survey team via
+      <a href='mailto:survey@emberjs.com'>survey@emberjs.com</a>.
     </p>
   </:body>
 </EmberCommunitySurvey::Section>
+
+{{!-- TODO (?) update to nicely accommodate multiple sponsors --}}
+<div class="bg-dark">
```

Comment:
We could add CSS grid to this `<div>` to show an arbitrary number of images side-by-side.

An example: https://github.com/ijlee2/ember-workshop/blob/main/app/styles/products.css#L48-L52